stagecoach/jobs/publishers/publish-all/app.py

```diff
@@ -65,23 +65,9 @@ async def _handler(assets):
         ]
     )
     _entries = await publisher_client.fetch()
-    response = await publisher_client.publish_many(_entries, pagination=50)
+    response = await publisher_client.publish_many(_entries, pagination=70)
```

Comment:
Any reason not to make this (70) the default in the client?

Reply:
default was 0 so that you only did pagination if opted in, i changed it to 80